test/utils/deployment.ts

```diff
@@ -152,6 +155,32 @@ async function confirmPostgresConnectivityAsync(maxTries: number = 5): Promise<v
         }
     }
 }
+async function confirmRedisConnectivityAsync(maxTries: number = 5): Promise<void> {
+    try {
+        await Promise.all([
+            // delay before retrying
+            new Promise<void>((resolve) => setTimeout(resolve, 2000)), // tslint:disable-line:custom-no-magic-numbers
```

Comment:
this rule drives me nuts

Reply:
yeah, not the biggest fan of it. Although it does do some good imo